app/Http/Requests/UpdateManagerProfileRequest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Services\Validation\Rules\ValidIdRule;
+use App\Models\Lookup\Frequency;
+use App\Models\Lookup\Department;
+use App\Services\Validation\Rules\LinkedInUrlRule;
+use Illuminate\Validation\Rule;
+use App\Services\Validation\Rules\PasswordCorrectRule;
+use App\Services\Validation\Rules\PasswordFormatRule;
+use App\Services\Validation\Rules\TwitterHandleRule;
+
+class UpdateManagerProfileRequest extends FormRequest
+{
+    /**
+     * Validator instance updated on failedValidation.
+     *
+     * @var \Illuminate\Contracts\Validation\Validator
+     */
+    public $validator = null;
+
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return boolean
+     */
+    public function authorize()
+    {
+        $manager = $this->route('manager');
+        return $manager->id !== null;
```

Comment:
1. If the Request class found a manager object, it did so using an id. That means that if `$manager` is not null, it definitely has an id. You'd want to check `$manager !== null`, not `$manager->id !==null`, which will either return true or throw an exception.
2. I don't think this check is necessary at all. Just by typehinting the Manager class in the controller route, it will fail if its null.